darts/models/regression_ensemble_model.py

```diff
@@ -51,7 +51,19 @@ def __init__(self,
         self.regression_model = regression_model
         self.train_n_points = regression_train_n_points
 
-    def fit(self, series: TimeSeries) -> None:
+    def fit(self, series: TimeSeries, epochs: Optional[int]=100) -> None:
```

Comment:
@hrzn this is a bit tricky because I went with a shortcut - you're unable to set different `epochs` value for any of the models in ensemble. Also `epochs` or `retrain_epochs`?

Reply:
I guess one way to avoid this could be to keep the `TorchForecastingModel.epochs` attribute, which is optionally set if a number of epochs is provided in `TorchForecastingModel.__init__`, but is overridden if `epochs` is provided in fit. That way we can still create an ensemble with models requiring different numbers of epochs. Just a random idea, there might be a better way..